streaming/kafka/cleanup.sh

```diff
@@ -5,24 +5,6 @@ set -o pipefail
 read -p "Enter the region: " region
 export AWS_DEFAULT_REGION=$region
 
-targets=(
```

Comment:
I think we need to keep this to ensure Kafka and add-ons are deleted first before deleting the EKS and VPC

targets=(
  "<Any kafka cluster resources>"
  "module.kubernetes_data_addons"
  "module.eks_blueprints_kubernetes_addons"
)

Reply:
Added addons modules